tests/record_file_descriptor.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+sleep 10
+IFS='
+';
+INITIAL_FILE_DESCRIPTOR_STORE=${INITIAL_FILE_DESCRIPTOR_STORE:-"initial.txt"}
+CURRENT_FILE_DESCRIPTOR_STORE=${CURRENT_FILE_DESCRIPTOR_STORE:-"final.txt"}
+
+for i in `ps xao pid,cmd | grep ocis | grep -v grep`
+do 
+  # echo $i
+  trimmed=$(echo $i | xargs)
+  PID=$(echo $trimmed | cut -d" " -f 1)
+  SERVICE=$(echo $trimmed | cut -d" " -f 3)
+  COUNT_DESCRIPTORS=`lsof -p $PID 2> /dev/null | wc -l`
```

Comment:
Codacy found an issue: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5973213)